tests/src/yktrace.rs

```diff
@@ -96,11 +96,11 @@ mod tir {
         assert!(tir_trace.len() > 0);
     }
 
-    struct DebugTirIO(usize, usize);
+    struct DebugTirInterpCtx(usize, usize);
```

Comment:
Interesting. Did we want `InterpCtx` across the board? More of these below.

Reply:
I don't think they all need to be `InterpCtx` if there's a better name. I don't know if `DebugTirInterpCtx` is a better name though.

Reply:
They don't need to all be the same name.

Reasonably, `ykbf` could use `BfInterpCtx` and I wouldn't argue.

Reply:
Fair enough, though that's not helping me decide whether `DebugTirInterpCtx` should stay as-is or not though :p

Reply:
Notice how it's defined outside of a test, to enable sharing. Well, we can't name all of the potentially existing structs shared in this way `InterpCtx`: the names would clash, so fine as is IMHO.

Reply:
OK.